.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
Do we really need separate testing for these three versions of ruby? At most I could see 2.2.x vs 2.3.x, but even those should be remarkably similar.